packages/shared-base/src/i18n/instance.ts

```diff
@@ -15,6 +15,8 @@ if (!i18n.isInitialized) {
     i18n.use(Detector).init({
         keySeparator: false,
         interpolation: { escapeValue: false },
+        pluralSeparator: '$',
+        contextSeparator: '$',
```

Comment:
how will it affect the use of plural and context?

Reply:
Existed code needs update.

I see only one use case, get it fixed.